org.endlessos.threadbare.yml

```diff
@@ -0,0 +1,34 @@
+# SPDX-FileCopyrightText: The Threadbare Authors
+# SPDX-License-Identifier: MPL-2.0
+id: org.endlessos.threadbare
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.godotengine.godot.BaseApp
+base-version: '4.4'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+finish-args:
+- --share=ipc
+- --socket=x11
```

Comment:
The editor has a "prefer wayland" mode that falls back to x11 if needed, but games apparently do not. Sad.